numba/unicode.py

```diff
@@ -622,6 +622,29 @@ def join_str_impl(sep, parts):
         return join_str_impl
 
 
+@overload_method(types.UnicodeType, 'zfill')
+def unicode_zfill(str_to_fill, width_to_fill):
+    def zfill_impl(fstr, fwidth):
```

Comment:
Lastly, you probably want to name the arguments the same as in the Python implementation, i.e. `width` instead of `fwidth`. Also maybe `string` instead of `fstr`.